crawlers/depodcast/depodcast_crawler.py

```diff
@@ -0,0 +1,58 @@
+import re
+import requests
+from dateutil.parser import parse
+from datetime import datetime
+from bs4 import BeautifulSoup
+from geopy.geocoders import Nominatim
+from ..base import BaseCrawler
+
+class DEPodcastCrawler(BaseCrawler):
+
+    def get_events(self):
+        geolocator = Nominatim(user_agent="Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/80.0.3987.149 Safari/537.36")
+        URL = 'https://www.dataengineeringpodcast.com/conferences/'
+        response = requests.get(URL)
+        soup = BeautifulSoup(response.content, 'html.parser')
+        results = soup.find(id='post-1401')
+        conf_list = results.find_all("section", class_='elementor-element')
+        for conf in conf_list:
+            title = conf.find('h2', class_='elementor-heading-title elementor-size-default')
+            place = conf.find('div', class_='elementor-custom-embed')
+            if title is not None and "Subscribe" not in title.text and place is not None:
+                event_info = title.text
+                hyp_index = event_info.index(' - ')
+                event_name_and_loc = event_info[:hyp_index]
+                event_dates = event_info[hyp_index+3:]
+                start_dt = event_dates.split(' - ')[0]
+                start_dt_formatted = datetime.strftime(parse(start_dt), '%Y-%m-%d')
+                month = start_dt.split(' ')[0]
+                end_dt = event_dates.split(' - ')[1] + ' ' +  month
+                end_dt_formatted = datetime.strftime(parse(end_dt), '%Y-%m-%d')
+                event_link = conf.find('a')['href']
+                place = place.iframe.extract().attrs['aria-label']
+                if "Convention Center" in place:
```

Comment:
This looks _very_ explicit. We need to make it more general. Can the `geolocator.geocode` call give us city, state and country?

Reply:
The reason I added this condition was for values of place like "David L. Lawrence Convention Center 1000 Fort Duquesne Boulevard Pittsburgh, PA 15222". When I pass this into geolocator.geocode(), it doesn't return any results. Any tips on how to handle this kind of a scenario?

Another doubt - If you go to the "https://www.dataengineeringpodcast.com/conferences/" link, now we see only virtual conference details present. In this case, will it be okay to print "Online" in city, state and country?

Reply:
Added further changes to retry in case of connection issues and addressed build failures due to methods. Awaiting your comments on the above doubt.

Reply:
Hi @vinayak-mehta, did you get a chance to look into this?

Reply:
Hi Vinayak, request you to kindly take a look at my questions and let me know. I am ready to implement and try any suggestions you have.

Reply:
Hey Kunal, sorry for being inactive here. I haven't been doing well: https://vinayak.io/now

This PR is not on high priority as https://www.dataengineeringpodcast.com/conferences doesn't have a lot of conferences, and doesn't update very often.